### ISSUE TITLE
Android project-scanner should respect include directive in settings.gradle

### ISSUE THREAD

**Issue by lhilden** [bitrise-steplib/steps-project-scanner#7]
Our Android repo has a mixture of active and obsolete sub-modules/projects in it.   The inactive sub-modules/projects are not actively maintained.  

```
build.grade
gradle.properties
settings.gradle
├── build
├── active_app
├── gradle
├── active_lib
├── active_app_2
├── inactive_app_1
└── inactive_app_2
```

Our top-level `build.gradle` file only includes the actively maintained sub-modules projects by explicitly listing them in the `include` directive in `settings.gradle` file like this:
```
include ':active_app', ':active_lib', ':active_app_2'
```

However, the Android project-scanner ignores the `settings.gradle` file and simply recursively scans the sub-module project directories for `build.gradle` files, even if those `build.gradle` files are invalid/no longer active.

This causes the "Validation Setup" step in "Add new app" to fail when it hits one of our inactive projects that has a bad `build.gradle` file. The workaround was simply to rename bad `build.gradle` files to `build.gradle.disabled.for.bitrise`

Ideally, the scanner would respect the `include` directive in the `settings.gradle` file, rather than picking up every `build.gradle` file in a subdirectory of the main project.

**Comment by godrei** [comment]
Hi @lhilden ,
thanks for reporting the issue!
The fix will be out today.

**Comment by godrei** [comment]
Hi @lhilden 
we released a new version of `project scanner` ([v0.9.4](https://github.com/bitrise-steplib/steps-project-scanner/releases/tag/0.9.4)), it inspect only the top-level `build.gradle` file(s).

Thanks for reporting!
